src/supercollider-js/SuperColliderJS.sc

```diff
@@ -48,19 +72,28 @@ SuperColliderJS {
 		});
 
 		thisProcess.nowExecutingPath = saveExecutingPath;
-		"SUPERCOLLIDERJS.interpreted".postln;
 		^""
 	}
 
 	*return { arg guid, type, object;
-		// post object as JSON to STDOUT
-		var json = this.stringify(object);
-		"\nSUPERCOLLIDERJS:%:START:%".format(guid, type).postln;
-		// sclang screws up when posting long lines in a single chunk
-		json.clump(2048).do { arg chunk;
-			"SUPERCOLLIDERJS:%:CHUNK:%".format(guid, chunk).postln;
-		};
-		"SUPERCOLLIDERJS:%:END:%".format(guid, type).postln;
+		if(resultSocket.isNil, {
+			"resultSocket must be connected before returning data".error;
+		}, {
+			var msg = (
```

Comment:
I think you could have just done `this.stringify(msg)`. Looks like you tried to do that but `()` is an Event in sc, not a simple dict.